aspects/electrical/Diode/test/UtGunnsElectRealDiode.cpp

```diff
@@ -364,35 +364,35 @@ void UtGunnsElectRealDiode::testConfirmSolutionAcceptable()
         tArticle->mPotentialVector[1] = 1.0;
         const GunnsBasicLink::SolutionResult expectedResult = GunnsBasicLink::CONFIRM;
         const bool                           expectedBias   = false;
-        const bool result = tArticle->confirmSolutionAcceptable(1, 1);
-        CPPUNIT_ASSERT(expectedResult == expectedResult);
+        const GunnsBasicLink::SolutionResult result = tArticle->confirmSolutionAcceptable(1, 1);
+        //CPPUNIT_ASSERT(expectedResult == result); FIXME: this assertion fails!
```

Comment:
@jasonlharvey What should we do here?

Reply:
Go ahead and submit this as you have it.  I wrote issue #122 to fix this test properly.  Good catch!